torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -1583,6 +1588,8 @@ c10::optional<ScheduleHeuristic> tryMerge(
     SegmentedGroup* b = nullptr) {
   FusionSegmentGuard fsg(fusion, getAllInputs(a, b), getAllOutputs(a, b));
 
+  scheduler_debug_utils::canScheduleMessage(
+      "\n**Segmenter** Considering fusion:\n", fusion);
```

Comment:
nit: Why the newline at the beginning?

Reply:
Just wanted this info to be more noticeable. Since this line means the segmenter is considering a new fusion graph.

Reply:
The printout will be quite verbose either way for a reasonably large fusion : )